.eslintrc.json

```diff
@@ -7,7 +7,10 @@
     "standard"
   ],
   "env": {
-    "node" : true
+    "node": true
+  },
+  "globals": {
+    "BigInt": true
```

Comment:
If you were to instead remove this new `globals` section and then add `"es2020": true` to the `env` section does it still run? I think that would be preferable.